docs/concepts/trading-on-vega/market-types.md

```diff
@@ -0,0 +1,72 @@
+:::
+
+## Perpetual futures
+Perpetual futures markets do not have a final settlement date and so traders can hold a position on a perpetuals market indefinitely. They periodically pay out profits to long or short traders by comparing the perpetual price to the underlying spot price.
+
+There's an extra settlement process for perpetual markets that ensures the perpetual price doesn't diverge too far from the spot prices of the trading pair. This is called a funding period, or periodic settlement. The schedule for the market's funding periods is set in its governance proposal.
+
+This scheduled settlement is in addition to mark to market settlement when the mark price changes.
+
+Perpetual futures markets have a settlement asset that may not be the same as either of the assets in the market's trading pair.
+
+### Funding rate
+Each market has a funding rate, which represents the difference between the mark price of the perpetuals market and the spot market price of the underlying asset. 
+
+If the perpetual price is above the spot price, the funding rate is positive and those with long positions "pay" those who with short positions. If the perpetual price is below the spot price, the funding rate is negative and traders with short positions lose money while those with long positions gain.
+
+Any profit or loss as the result of the funding rate is only final once the position is closed, and is tracked as unrealised P&L.
+
+The funding rate formula is as follows:
+
+`funding rate = funding payment / spot time-weighted average price (TWAP)`
+
+Read more on the funding payment below.
+
+:::tip Query for data
+See a market's historical funding rates using the [REST API](../../api/rest/data-v2/trading-data-service-list-funding-periods.api.mdx).
+:::
+
+### Funding payment
+The funding payment is, in its simplest form, calculated as the difference beween the perpetual market's current time-weighted average price and the external spot time-weighted average price. The perpetuals price is based on the mark price on the market, while the spot price comes from the data source.
+
+Perpetuals market proposers can include additional parameters into the funding payment calculations to more closely match existing popular markets.
+
+They include:
+* Interest rate
+* Clamp lower bound
+* Clamp upper bound
+
+:::note Go deeper
+**[Funding payment calculations ↗](https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/0053-PERP-product_builtin_perpetual_future.md#funding-payment-calculation)**: See how all the parameters can impact a market's funding rate and margin requirements.
+:::
+
+### Margin calculations
+The margin calculations for perpetual futures markets contain an additional parameter to capture the exposure of a given position to an upcoming funding payment. The margin funding factor that determines to what degree the funding payment amount impacts a trader's maintenance margin is also defined in the market proposal.
```

Comment:
```suggestion
The margin calculations for perpetual futures markets contain an additional parameter to capture the exposure of a given position to an upcoming funding payment. The margin funding factor that determines to what degree the funding payment amount impacts a trader's maintenance margin is also defined in the market proposal. This additional term can only increase the margin requirement if a given position is expected to make a payment at the end of the current funding period, but it will never decrease the margin requirement, even if party is expecting to receive a funding payment.
```